api/config/config_local.json

```diff
@@ -74,7 +74,7 @@
     "enableMoodleBackend": true
   },
   "access": {
-    "enableComplianceTraining": false,
+    "enableComplianceTraining": true,
```

Comment:
Enabled this so I can actually test these changes. 'bypass all' should still work.